airflow/models/dagrun.py

```diff
@@ -498,6 +498,37 @@ def fetch_task_instances(
             tis = tis.where(TI.task_id.in_(task_ids))
         return session.scalars(tis).all()
 
+    @internal_api_call
+    def _check_last_N_dagruns_failed(
```

Comment:
```suggestion
    def _check_last_n_dagruns_failed(
```

Just keep everything lowercase